Tests/BodegaTests/DiskStorageEngineTests.swift

```diff
@@ -1,12 +1,13 @@
 import XCTest
 @testable import Bodega
 
-final class DiskStorageTests: XCTestCase {
+final class DiskStorageEngineTests: XCTestCase {
 
-    private var storage: DiskStorage!
+    private var storage: DiskStorageEngine!
 
     override func setUp() async throws {
-        storage = DiskStorage(directory: .temporary(appendingPath: "Tests"))
+        storage = DiskStorageEngine(directory: .temporary(appendingPath: "Tests"))
```

Comment:
just out of curiosity if you run the tests with parallelism/randomness do they still pass. I'm just wondering if the returned "temp directory" differs with each test. otherwise you may get some test pollution. I suspect the `removeAllData` may be keeping it at bay assuming theres no parallelism

Reply:
I turned on parallelism and randomness and the tests still pass. The temporary directory is actually `tmp` each time, so pretty standard.
```
file:///Users/mergesort/Library/Developer/XCTestDevices/UUID1234-UUID-UUID-UUID-UUID12345678/data/tmp/Tests/
```

Reply:
ahh sweet, it throws a UUID in there. should be good to go then 🙌

Reply:
Kind of, I should have been clearer. The UUID is just the simulator ID, which is a UUID. I just wrote it that way cause that's where my brain went. 😅